core/local/chokidar_watcher.js

```diff
@@ -326,12 +326,14 @@ module.exports = class LocalWatcher {
           case 'FileMove':
             if (c.needRefetch) {
               c.old = await this.pouch.db.get(metadata.id(c.old.path))
-              c.old.childMove = false
             }
             await this.onMoveFile(c.path, c.stats, c.md5sum, c.old, c.overwrite)
             if (c.update) await this.onChange(c.update.path, c.update.stats, c.update.md5sum)
             break
           case 'DirMove':
+            if (c.needRefetch) {
+              c.old = await this.pouch.db.get(metadata.id(c.old.path))
```

Comment:
Oh, the root of the 2nd move didn't had its old metadata refreshed after applying the first one :ok_hand:

Reply:
Yes, since child moves are created when merging the parent dir move, we're passed the prep phase of the second move.